### PR TITLE
Config

### PR DESCRIPTION
**Pull request by vuillaut**
Is the config clearer this way?
- The parameters that do not differ from the standard config are not repeated in derived ones.
- the MC image extractor parameters are clearly specified and separated from the muons ones

Solves #422

### CODE REVIEW

File: lstchain/data/lstchain_standard_config.json [hunk 1]
```diff
@@ -90,70 +85,95 @@
...
+    3,
+    4
+  ],
   "max_events": null,
   "custom_calibration": false,
   "write_pe_image": false,
   "image_extractor": "LocalPeakWindowSum",
-  "image_extractor_for_muons": "GlobalPeakWindowSum",
-  "image_extractor_config": {},
+  "image_extractor_config": {
```
Comment (MaxNoe) [inline lstchain/data/lstchain_standard_config.json:98]: Why are both configs given here if only `LocalPeakWindowSum` is used?

File: lstchain/data/lstchain_standard_config.json [hunk 2]
```diff
@@ -90,70 +85,95 @@
...
-
-  "LocalPeakWindowSum":{
-     "window_shift": 4,
-     "window_width":8
-    },
-  "GlobalPeakWindowSum":{
-     "window_shift": 4,
-     "window_width":8
+  "image_extractor_for_muons": "GlobalPeakWindowSum",
+  "image_extractor_for_muons_config": {
```
Comment (MaxNoe) [inline lstchain/data/lstchain_standard_config.json:113]: Same as above. Why not only give the config for the `GlobalPeakWindowSum`?

File: lstchain/data/lstchain_standard_config.json (hunk 1 above)
Comment (vuillaut) [reply lstchain/data/lstchain_standard_config.json:98]: Not need, may (or not) be more clear to be exhaustive on the possible configs, for example if the user wants to switch the `image_extractor`, the parameters are set.
Feedback welcome.

Comment (MaxNoe) [reply lstchain/data/lstchain_standard_config.json:98]: `--help-all` should show all possible settings for tools.

Comment (vuillaut) [reply lstchain/data/lstchain_standard_config.json:98]: > `--help-all` should show all possible settings for tools.

Nice.
For that we would need to make Tools out of all scripts.

Comment (vuillaut) [reply lstchain/data/lstchain_standard_config.json:98]: I'd be happy to limit the standard config to only the necessary one if you think it's clearer (I do) - it wasn't, so I kept it this way.

Comment (MaxNoe) [reply lstchain/data/lstchain_standard_config.json:98]: @vuillaut that is needed anyways to get to a consistent api and to be able to properly use the configuration system. Currently, this all fells really hacked together around some of the limitation the ctapipe config system has (or had, a lot are fixed in 0.8)

**Comment by vuillaut** [comment]
I think this should be considered review, we still are unable to configure the image extractor parameters for MC in the present config.
If there is something that needs to be changed, please let me know.

File: lstchain/data/lstchain_standard_config.json (hunk 2 above)
Comment (vuillaut) [reply lstchain/data/lstchain_standard_config.json]: For this PR I'd like to simply fix the config bug.
The whole config will need to be changed and improved with the use of tools and ctapipe0.8.

**Comment by vuillaut** [comment]
After lstchain 0.6 release, it was easier to open a new PR #496

**Pull request closed without merging**